cli/dcoscli/data/help/marathon.txt

```diff
@@ -92,6 +95,15 @@ Commands:
         Display detailed information for a specific pod.
     pod update
         Update a pod.
+    debug list
+        Print a list of currently queued instance launches for
+        debugging purpose.
+    debug summary
+        Display summarized information for a queued instance launch
+        for debugging purpose.
+    debug details
+        Display detailed information for a queued instance launch
+        for debugging purpose.
```

Comment:
We're trying to get the command to match the UI. Right now, we've put it under a `debug` tab in the UI and so it should be something similar here.